lib/std/fs/wasi.zig

```diff
@@ -131,18 +156,44 @@ pub const PreopenList = struct {
                 else => |err| return os.unexpectedErrno(err),
             }
             const preopen_len = buf.u.dir.pr_name_len;
-            const path_buf = try self.buffer.allocator.alloc(u8, preopen_len);
+            var path_buf = try self.buffer.allocator.alloc(u8, preopen_len);
```

Comment:
Is this change needed here? I don't see you re-assigning `path_buf` anywhere. Note that `allocator.alloc` returns a pointer so you can still do interior mutability on a `const`.

Reply:
Oops, this is a leftover from some hacking - I'll clean it up!